qemu/tests/hugepage_mem_stress.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu hugepage memory stress test.
+    Steps:
+    1) System setup hugepages on host.
+    2) Mount this hugepage to /mnt/kvm_hugepage.
+    3) Run memory heavy stress inside guest.
+    4) Check guest call trace in dmesg log.
+
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+    session = vm.wait_for_login()
+    stress_args = params["stress_cmd"] % int(float(params.get("mem")) / 512)
+
+    error_context.context("Run memory heavy stress in guest", logging.info)
+    stress_test = utils_test.VMStress(vm, "stress", params,
+                                      stress_args=stress_args)
+    try:
+        stress_test.load_stress_tool()
+        stress_test.unload_stress()
+        utils_misc.verify_dmesg(session=session)
+    finally:
+        stress_test.clean()
```

Comment:
Suggest that add a step to shut down the guest to check whether the guest is hung, according to the manual case's steps. It just up to you and feature owner.
Thanks.